qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -450,23 +441,51 @@ void allowsOnlyProperlyOrderedAtClauses() throws Exception {
      * @throws Exception If something wrong happens inside
      */
     @Test
-    @Disabled
-    void passesWindowsEndsOfLineWithoutException() throws Exception {
-        this.validate("WindowsEol.java", false, "LICENSE found:");
+    void prohibitWindowsEndsOfLine() throws Exception {
+        final String file = "WindowsEol.java";
+        final Collection<Violation> results = this.runValidation(file, false);
+        final String message = "Lines in file should end with Unix-like end of line";
+        final String name = "RegexpMultilineCheck";
+        MatcherAssert.assertThat(
+            results,
+            Matchers.contains(
+                new ViolationMatcher(
+                    "Expected line ending for file is LF(\\n), but CRLF(\\r\\n) is detected",
+                    file, "1", "NewlineAtEndOfFileCheck"
+                ),
+                new ViolationMatcher(
+                    message, file, "9", name
+                ),
+                new ViolationMatcher(
+                    message, file, "10", name
+                )
+            )
+        );
     }
 
     /**
      * Fail validation with Windows-style formatting of the license and
      * Linux-style formatting of the sources.
      * @throws Exception If something wrong happens inside
-     * @todo #61:30min This test and passesWindowsEndsOfLineWithoutException
-     *  should be refactored to gather log4j logs and validate that they work
-     *  correctly. (see changes done in #61)
      */
     @Test
-    @Disabled
+    @EnabledOnOs(OS.WINDOWS)
     void testWindowsEndsOfLineWithLinuxSources() throws Exception {
-        this.runValidation("WindowsEolLinux.java", false);
+        final String file = "WindowsEolLinux.java";
+        final Environment.Mock mock = new Environment.Mock();
+        final File license = this.createLicense(mock, "\r\n", "Hello.", "World.");
+        final Environment env = this.configureEnvironment(mock, license, file);
+        final Collection<Violation> results = new CheckstyleValidator(env).validate(
+            env.files(file)
+        );
+        final String message = "Line does not match expected header line of ' *'";
+        final String name = "HeaderCheck";
+        MatcherAssert.assertThat(
+            results,
+            Matchers.contains(
+                new ViolationMatcher(message, file, "3", name)
```

Comment:
A can not match test name and validation in it: 
test is about windowEndsOfLine
validation is about HeaderCheck.